com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -596,6 +612,7 @@ half4 UniversalFragmentPBR(InputData inputData, half3 albedo, half metallic, hal
     for (uint lightIndex = 0u; lightIndex < pixelLightCount; ++lightIndex)
     {
         Light light = GetAdditionalLight(lightIndex, inputData.positionWS);
+        light.shadowAttenuation = ApplyShadowFade(light.shadowAttenuation, inputData.positionWS);
```

Comment:
shadow fade should only be applied to main directional light as this is controlled by shadow distance. additional lights should not apply it.